packages/design-system-docs/src/pages/startup/components.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Using components
+weight: 3
+---
+
+<h2>Using React components</h2>
+
+**We strongly recommend using the React versions of our components.** It allows for more complex, interactive components like `Tabs` or `VerticalNav`, and many of our React components include accessibility optimizations.
+
+The examples below assume you've [installed the design system using NPM]({{root}}/startup/installation/) and have already setup your build system for React.
+
+<h3>Named imports</h3>
+
+Components can be imported from the package entry point using the syntax below.
+
+<div class="ds-c-alert ds-c-alert--warn ds-u-margin-bottom--2 ds-u-font-size--small">
+  <div class="ds-c-alert__body">
+    <h3 class="ds-c-alert__heading ds-u-font-size--base">Performance note</h3>
+    <p class="ds-c-alert__text">
+      This approach may result in a larger file size than intended. Depending on what module bundler you use, all of the design system's React components may be included in the bundled file even if you didn't specifically import them. This can be avoided by using [tree shaking in Webpack](https://webpack.js.org/guides/tree-shaking/).
+    </p>
+  </div>
+</div>
+
+```jsx
+import { Button } from '@cmsgov/design-system';
+import { TextField, ChoiceList } from '@cmsgov/design-system';
+```
+
+The `@cmsgov/design-system` package is configured to automatically work with Webpack 4's tree shaking. We provide a separate entry point (`index.es.js`) and component directory (`esnext`) for ES modules to support these optimizations.
```

Comment:
Should we say that the resolving of this `index.es.js` module specifically is what's automatic? I read this as the tree-shaking in webpack working automatically, but it sounds to me like I have to import that `index.es.js` file directly.

Reply:
What about something like this? I just added on to that last sentence to explain.
> The `@cmsgov/design-system` package is configured to automatically work with Webpack 4's tree shaking. We provide a separate entry point (`index.es.js`) and component directory (`esnext`) for ES modules and configure our package so that Webpack will use this entry point when _ES Modules_ are preferred.

Reply:
Maybe I'm going in circles. I just don't want them thinking that if they import `@cmsgov/design-system/dist/components` that it will somehow magically change that path into the ES modules one

Reply:
thats definitely more clear, I'll make that change

Reply:
Great! Other than this, it all LGTM 👍

Reply:
I went with something a little more concise. I kept the last sentence, and removed the word "automatic" to make it seem less magic

> The `@cmsgov/design-system` package provides an ES modules version of our code (located in `dist/esnext`) to support Webpack 4's tree shaking optimizations. Our package is configured to let Webpack know to resolve imports from `"@cmsgov/design-system"` to the ES modules version when preferred.

Reply:
Yeah, that's better!

Reply:
Oops, I thought i pushed this change up before merging but I didnt, I'll get it in the follow up